Rubberduck.Parsing/VBA/RubberduckParserState.cs

```diff
@@ -584,7 +584,8 @@ public void ClearStateCache(VBProject project, bool notifyStateChanged = false)
 
             if (notifyStateChanged)
             {
-                OnStateChanged();
+                OnStateChanged(ParserState.ResolvedDeclarations);   // trigger test explorer and code explorer updates
+                OnStateChanged(ParserState.Ready);   // trigger find all references &c. updates
```

Comment:
CE and TE only refresh when ResolvedDeclarations fires.  Other things only refresh when Ready fires.  I need both of these to be fired.